src/Sulu/Bundle/AdminBundle/Resources/js/components/AutoCompletePopover/tests/AutoCompletePopover.test.js

```diff
@@ -158,9 +158,8 @@ test('Pressing up should select previous item', () => {
     const suggestionElement1 = {focus: jest.fn()};
     const suggestionElement2 = {focus: jest.fn()};
 
-    // $FlowFixMe
     Object.defineProperty(document, 'activeElement', {
-        value: suggestionElement2,
+        value: (suggestionElement2: any),
```

Comment:
I would personally make more use of `// $FlowFixMe` instead of casting everything to `any`. With `any` we are losing all typechecks that are happening afterwards, and I think with `$FlowFixMe` we are only fixing this error. Especially since it seems that they also want to support suppressing only certain error codes.

And it still works when moving the comment above this line directly.

Reply:
you are right, will move the `$FlowFixMe`